分類基準（区分原理）の変更

eniehack opened the pull request:
分類法を変更
related: #6 

全体に対して:
リストでは階層構造を作らない

ツールに関して:
1. 公式の情報源、ツールか否か？公式のものでなければ2へ
2. twinsやKdBなどサービスに関連するもので分類、できない場合は3へ
3. 「履修スケジュール確認」などのシチュエーション別の分類、できない場合は4へ
4. 「その他」として分類

関連する要素の統廃合

eniehack committed:
Update README.md

分類法を変更

全体に対して:
リストでは階層構造を作らない

ツールに関して:
1. 公式の情報源、ツールか否か？公式のものでなければ2へ
2. twinsやKdBなどサービスに関連するもので分類、できない場合は3へ
3. 「履修スケジュール確認」などのシチュエーション別の分類、できない場合は4へ
4. 「その他」として分類

関連する要素の統廃合

onokatio commented:
@eniehack 
`Lists should be surrounded by blank lines`, `Headings should be surrounded by blank lines`だけ修正してもらってもいいですか？

eniehack commented:
わかりました！

eniehack committed:
Update README.md

fix `Unordered list indentation`, `Headings should be surrounded by blank lines`, `Lists should be surrounded by blank lines`

onokatio commented:
ありがとうございます！

eggplants commented:
`MD024` を[mdl-config.json](https://github.com/Make-IT-TSUKUBA/awesome-tsukuba/blob/main/.github/mdl-config.json)で黙らせるか、見出しの重複を排してもらえると…
